### PR TITLE
Feature/config email testing on local environment

### PR DESCRIPTION
**Pull request by 511234**
Ref: #137

Should we also cc the sender so they could keep a record of what they've sent?

### CODE REVIEW

**Review by yaucp — DISMISSED** [review]

**Review by Pjaijai — APPROVED** [review]
Cool

**Comment by Pjaijai** [comment]
Yea I think cc sender is a good idea

File: DEVELOPER.md [hunk 1]
```diff
@@ -68,7 +68,11 @@ create or replace trigger on_auth_user_created
 5. Run `supabase link --project-ref [your-supabase-project-id]`
 6. Run `supabase db push` to run migrations on your supabase project
 7. Import [csv files](https://drive.google.com/drive/folders/14Q5xqmHU9w1v7Zv9HlxL3qr--Cw_4z7b?usp=drive_link) into your Supabase project for `city`, `country`, `industry` and `province` tables
-8. Connect Supabase database to your favourite database IDE. Run [SQL Script](#to-solve-user-data-cannot-be-inserted-after-registration) to insert user data upon registration
+8. Connect Supabase database to your favourite database IDE. Run [SQL Script](#to-solve-user-data-cannot-be-inserted-after-registration) to insert user data upon registration 
+9. Clone `.env.template` from `supabase/` and rename it to `.env`
+10. Create a [Resend](https://resend.com) account for email sending. Link it to your Supabase project and get an API key.
+11. Configure your `.env`, add your `RESEND_API_KEY` from Resend and use your Resend email account as `RESEND_TO_EMAIL`.
+12. Every time you make changes to Supabase functions, run `supabase functions deploy --project-ref [your-supabase-project-id]`
```
Comment (Pjaijai) [inline DEVELOPER.md:75]: For deploying function, we have some yml files to do github actions. The edge function shell auto deploy when merging to development and main branch

File: supabase/.env.template [hunk 2]
```diff
@@ -1 +1,3 @@
-RESEND_API_KEY=
\ No newline at end of file
+RESEND_API_KEY=
+TO_EMAIL=
```
[the remark this answers is not in the document]
Comment (511234) [reply supabase/.env.template:2]: No need, just make sure your .env contains `ENVIRONMENT` & it's set to `PRODUCTION` / `STAGING`
* btw thanks for pointing this out, I just realize I have not updated the variable in `.env.template`

File: DEVELOPER.md (hunk 1 above)
Comment (511234) [reply DEVELOPER.md:75]: sure, when people are developing on feature branch, they should be able to perform manual deployment.

**Review by Pjaijai — APPROVED** [review]
Many Thanks

**Pull request merged**